build.gradle

```diff
@@ -193,6 +193,7 @@ task slimJar(type: Jar) {
     include 'fitnesse/socketservice/*.class'
     include 'util/*.class'
     include 'fitnesse/util/StringUtils.class'
+    exclude 'fitnesse/slim/converters/beans/*.class'
```

Comment:
I'm not sure whether this is needed. Since we access via reflection, and ignore exceptions, we might be able to include this class and still have everything work on android. Including it seems better as it would allow jar to be used on java also, with PE support. Can you check?

Reply:
when I remove this `exclude`, the jar is still fine for Android